miio/airhumidifier.py

```diff
@@ -247,6 +261,10 @@ def __init__(
         # TODO: convert to use generic device info in the future
         self.device_info: Optional[DeviceInfo] = None
 
+        _LOGGER.warning(
+            "The 'depth' property is deprecated and will be removed for 3 months (october 2021). Use 'water_level' and 'water_tank_detached' properties instead."
```

Comment:
```suggestion
            "The 'depth' property is deprecated and will in the future. Use 'water_level' and 'water_tank_detached' properties instead."
```
Move this to the `depth` property itself so it won't cause a warning if one is not using it. Also, no need to be that specific on when it gets removed, just that it's deprecated. There is also `deprecated` decorator available in utils that can be used (which uses warnings instead of logger for warning).